loky/process_executor.py

```diff
@@ -831,3 +848,18 @@ def shutdown(self, wait=True):
     shutdown.__doc__ = _base.Executor.shutdown.__doc__
 
 atexit.register(_python_exit)
+
+
+class _ExecutorFlags(object):
+    """necessary references to maintain executor states without preventing gc
+
+    It permits to keep the information needed by queue_management_thread
+    and management_thread to maintain the pool without preventing the garbage
+    collection of unreferenced executors.
+    """
+    def __init__(self):
+
+        # queue state
```

Comment:
This comment is off. Please remove it and just keep the docstring of the class (it's informative enough). Feel free to push a fix with `[ci skip]` in the commit message to not trigger CI again for comment fixes.